sandbox-web/src/main/java/com/exadel/sandbox/team5/CompanyRestController.java

```diff
@@ -48,6 +49,13 @@ public CompanyDto update(@PathVariable Long id, @RequestBody CompanyDto company)
         return companyService.update(company);
     }
 
+    @PutMapping("/{id}/uploadImage")
+    public CompanyDto updateImage(@PathVariable Long id, @RequestBody MultipartFile file) {
+        CompanyDto company = companyService.getById(id);
+        company.setImageId(imageService.save(file));
+        return companyService.update(company);
```

Comment:
It is better to create a separate method `companyService.updateImage(id, imageDto)`. The similar method for discounts.